css/index.css

```diff
@@ -143,4 +143,158 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* sprint challenge
+
+html {
+    font-size:62.5%;
+}
+*/
+
+
+a{
+    text-decoration: none;
+    color: black;
+}
+
+/* Desktops and laptops ----------- */
+@media only screen  and (min-width : 1000px) and (max-width : 2000px) {
+    /* Styles */
+  
+
+   nav {
+       display: flex;
+       justify-content: space-between;
+       padding: 20px 10px;
+   }
+
+   ul {
+       display: flex;       
+   }
+
+   .middle-content .boxes {
+       width: 920px;
```

Comment:
Should be `width: 100%`. 920px makes the boxes extend the width of the outer container (which is set to 800px).